app/src/ui/lib/configure-git-user.tsx

```diff
@@ -19,9 +19,6 @@ interface IConfigureGitUserProps {
 
   /** The label for the button which saves config changes. */
   readonly saveLabel?: string
-
-  /** Additional elements that are rendered below the form. */
-  readonly children?: ReadonlyArray<JSX.Element>
```

Comment:
I was hoping to keep these around but any `children` member is ignored because it exists on `React.Component<P, S>`:

```
props: Readonly<{ children?: ReactNode }> & Readonly<P>;
```